tests/perl/tests.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+
+# Copyright 2017 tsuru authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+setup() {
+    rm -rf /home/application/current && mkdir /home/application/current
+    chown ubuntu /home/application/current
+    export CURRENT_DIR=/home/application/current
+}
+
+@test "check if carton is intalled" {
+    pushd ${CURRENT_DIR}
+    run carton -v
```

Comment:
Yea, you need to do like in the python example and write the cpanfile yourself (the example app is not on the units tests). I think its ok to rely on internet access for those (others are).